scd30/Makefile

```diff
@@ -22,7 +22,7 @@ scd_git_version.o: ${scd_common_dir}/scd_git_version.c
 	$(CC) $(CFLAGS) -c -o $@ $^
 sensirion_common.o: ${sensirion_common_dir}/sensirion_common.c
 	$(CC) $(CFLAGS) -c -o $@ $^
-sensirion_sw_i2c_implementation.o: ${sw_i2c_dir}/sample-implementations/linux_user_space/sensirion_sw_i2c_implementation.c
```

Comment:
This isn't really related to the strict aliasing. But since it is in separate commit I think I'm OK with it :wink:

Reply:
I'll merge linearly so it shouldn't be visible in the history